### DISCUSSION TITLE
There is a typo in the main `__init__.py`

### DISCUSSION THREAD

**Discussion by airatk** [sqlalchemy/sqlalchemy#10578] [Usage Questions] [answered]
There is a typo in the `lib/sqlalchemy/__init__.py` file — it should be a <code>Single**t**onThreadPool</code> instead of `SingleonThreadPool`, the **t** letter is missing. 
https://github.com/sqlalchemy/sqlalchemy/blob/b51cccec6a953555f39c16005cb5a2a49a6f4b21/lib/sqlalchemy/__init__.py#L58

**Comment by CaselIT** [comment]
hi,

thanks for reporting it

**Reply by CaselIT** [reply, marked as answer]
fixed in https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4961
